keychain.h

```diff
@@ -45,23 +48,109 @@ class QKEYCHAIN_EXPORT Job : public QObject {
     QSettings* settings() const;
     void setSettings( QSettings* settings );
 
+    /**
+     * Call this method to start the job.
+     * Tipically you want to connect some slot to the finished() signal first.
+     * You can run the job either synchronously or asynchronously.
+     *
+     * In the first case you tipically use an inner event loop:
+     *
+     * \code
+     * SomeClass::startJob()
+     * {
+     *     QEventLoop eventLoop;
+     *     connect(job, &Job::finished, &eventLoop, &QEventLoop::quit);
+     *     job->start();
+     *     eventLoop.exec();
```

Comment:
Fair enough (I mentioned them only because I saw they are used in `testclient.cpp` :p)